test/current-color.scss

```diff
@@ -0,0 +1,3 @@
+.test {
+    color: currentColor;
```

Comment:
https://css-tricks.com/currentcolor/ cant we use currentcolor? but yeah seems good for me :)

Reply:
we can but it looks just false for me 😆

Reply:
The spec also reference it as `currentColor`: https://www.w3.org/TR/css-color-3/#currentcolor